app/javascript/components/patient/assessment/actions/MonitoringPeriod.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
```

Comment:
I love breaking this out!!! Can you add a front end test file and tests for this?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/ea0697f83c29320b8930fb7a5dcf63015ea342fb#diff-e15f1c6b962325b3f9d37b9159f62060066188401f072899685088ff987b792bR1